src/components/tearComponent/index.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  ResponsiveImg,
+  Background,
+  StyledExternalWrapper,
+  StyledInternalWrapper,
+} from './index.styles'
+
+import { Tear } from './tears'
+
+const TornBackground = ({
+  backgroundImage,
```

Comment:
The image should be part of the children, rather than as a `prop`. This leads to more flexibility within the component.